src/vsr/client_replies.zig

```diff
@@ -148,8 +157,6 @@ pub fn ClientRepliesType(comptime Storage: type) type {
         ) ?*Message.Reply {
             const client = session.header.client;
 
-            if (!client_replies.writing.isSet(slot.index)) return null;
```

Comment:
We had this fast-path here from the begining, but I think its wrong, due to 

```
                var reads = client_replies.reads.iterate();
                while (reads.next()) |read| {
                    if (read.slot.index == write.slot.index) return;
                }
```

loop in write